src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:22454/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:22454/project/issues?id=sample&branch=feature1&issues=AWFGIMRI3SJ0OPJke-9J&open=AWFGIMRI3SJ0OPJke-9J)

[ISSUE_KEY]: AWFGIMRI3SJ0OPJke-9J